GCEWindowsAgent/main.go

```diff
@@ -64,6 +64,23 @@ func writeSerial(port string, msg []byte) error {
 	return nil
 }
 
+func readSerial(port string) ([]byte, error) {
```

Comment:
i thought the trigger would be network or metadata. who will write to serial port, and do we have precedent for that?